src/components/Wrapper/Wrapper.js

```diff
@@ -25,7 +25,7 @@ export default ({ children, ...restProps }) => {
         <Navigation authService={authService} />
         <Content>
           {/* should render the role and path on the main content rendering area */}
-          <LocationInfo>
+          {/* <LocationInfo>
```

Comment:
are we commenting this out?? or was this left over from previous debugging session?

Reply:
we are uncommenting this, it seems to be working correctly